### ISSUE TITLE
Add Unity

### ISSUE THREAD

**Issue by lex3001** [xpdanet/xpdanet.github.io#90]

**Comment by TheJaredWilcurt** [comment]
Yeah, unitiy is a big one. But I'd need to do a lot more research on it, as it isn't just a singular tool or framework, but a suite of different products for different uses. Not sure how it fits in with the rest of the tools on the list. Though they've started to branch out, they're still primarily known as a game engine.
